bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -540,11 +556,16 @@ class VideoPreview extends Component {
                      onChange={this.handleSelectProfile}
                      disabled={skipVideoPreview}
                    >
-                     {availableProfiles.map(profile => (
+                     {availableProfiles.map(profile => { 
+                      const label = intlMessages[`${profile.id}`] 
+                        ? intl.formatMessage(intlMessages[`${profile.id}`])
+                        : profile.name;
+
+                      return (
                        <option key={profile.id} value={profile.id}>
-                         {profile.name}
+                          {`${label} ${profile.id === 'hd' ? '' : intl.formatMessage(intlMessages.qualityLabel).toLowerCase()}`}
```

Comment:
This breaks the German language, as the quality label should be Capital case.
Why not simply use the name of the profile as it was before? Or, add an extra label field to the camera profiles?

Reply:
I see, localization isn't possible the old way. But like the previous speaker ( @mvasylenko ), I am against simply appending a string to the name. We are using custom camera profiles and it breaks the naming.